### DISCUSSION TITLE
Dashy under Proxmox - Unable to get local-icons working

### DISCUSSION THREAD

**Discussion by TimeJunkie01** [tteck/Proxmox#1065] [Help Needed]
Dashy setup is a breeze using your script(s) on Proxmox. I have pages of widgets, and multiple icons fully working. Just not local icons.

I am unable to set dashy to use local icons. fas/hl etc all work.

I have tried (under /dashy) the following directories

|-- icons
|   `-- dashboard-icons
|       |-- png
|       `-- SVG

(same for under public)

|-- icons
|   `-- dashboard-icons
|       |-- png
|       `-- SVG

using: public/icons/dashboard-icons/png/proxmox.png

Do you have any tips for this?

**Comment by TimeJunkie01** [comment]
[11:49:57] The path to 'icons/dashboard-icons/png/proxmox.png' could not be resolved
[11:54:34] The path to 'icons/dashboard-icons/png/proxmox.png' could not be resolved
Directories are fully rwx

**Comment by Nivin37** [comment]
I have this same issue, anyone have an answer, or a direction to look?
